.github/CONTRIBUTING.md

```diff
@@ -1,28 +1,38 @@
-5. [Create a new branch](https://github.com/firstcontributions/first-contributions/blob/main/README.md#create-a-branch) from the `master` branch.
-6. [Create, commit, and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes. Make sure you're committing to your new branch.
-7. Go to the website and enable workflows in Actions on your fork so we can automatically clean up your code if necessary.
-8. Finally, [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). We will review your changes and talk about any further improvements if necessary.
-
-(These instructions are for using [Git](https://git-scm.com/) to commit your changes, but you can also use the website, [GitHub Desktop](https://desktop.github.com/), [Visual Studio Code](https://code.visualstudio.com/), or any other software that lets you make your changes.)
+3. If you already have a fork, make sure to [sync its `master` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to ensure your fork of Scratch Addons is up to date with the source code.
+4. [Clone your fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) and [load it into your browser](https://github.com/ScratchAddons/ScratchAddons#from-source) so you can make and test your changes. If you've already cloned your fork, remember to [pull down any changes](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#fetching-changes-from-a-remote-repository) to make sure you're up to date.
+5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch.
+6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to your new branch.
+7. If you haven't already, go to your fork on the website and enable workflows in Actions on your fork. This will enable an automation that formats (cleans up) your code if necessary.
+8. Finally, go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. (If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Click "create pull request" to go to the next step.) Finally, fill out the form and click "create pull request" to finish! We will review your changes and talk about any further improvements if necessary.
```

Comment:
Super picky review (safe to ignore): Separate this to multiple numbers, or a sublist?